backend/src/index.js

```diff
@@ -125,6 +125,65 @@ app.get(
   })
 )
 
+app.get(
+  '/set-status/:status/:id',
+  errorWrap(async (req, res) => {
+    try {
+      let response = "Status set Sucessfully"
+      switch(req.params.status) {
+        case "pending":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "pending"});
+          break;
+        case "accepted":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "accepted"});
+          break;
+        case "interviewing":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "interviewing"});
+          break;
+        case "rejected":
+          await Candidate.findByIdAndUpdate(req.params.id, {status: "rejected"});
+          break;
+        default:
+          response = "Invalid status, please try again"
+      }
+      res.json({result: response})
+    } catch (err) {
+      res.status(400).json({ message: err.message })
+    }
```

Comment:
Sorry for providing a bad example previously, but `errorWrap` catches any errors within the async function. This is what #41 is doing. errorWrap would catch the error and the errorHandler will handle, thus, removing the need for try...catch blocks. Let's remove the try...catch blocks for all of your new and modified endpoints. #41 removes try...catch blocks for the previously written endpoints.